torchvision/tv_tensors/_bounding_boxes.py

```diff
@@ -111,6 +116,7 @@ def __new__(
         requires_grad: bool | None = None,
     ) -> BoundingBoxes:
         tensor = cls._to_tensor(data, dtype=dtype, device=device, requires_grad=requires_grad)
+        cls._check_format(tensor, format=format)
```

Comment:
Since we call it only once and it's only 2 lines, we can inline it instead of making it a method. Also it might be best to do the validation before as the very first step, before calling `cls._to_tensor()`?

Reply:
@NicolasHug, I included it within the function. However, it possible to pass list as input so it easier to run this test after the input has been converted to a tensor.